### PR TITLE
Azure Arc-enabled data services - Adding bastion

### PR DESCRIPTION
**Pull request by sebassem**
closing #987

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_dc_vanilla_arm_template/_index.md [hunk 1]
```diff
@@ -139,16 +141,18 @@ As mentioned, this deployment will leverage ARM templates. You will deploy a sin
 
 ## Windows Login & Post Deployment
 
-- Now that first phase of the automation is completed, it is time to RDP to the client VM using it's public IP.
+- Now that first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using it's public IP.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/aks/aks_dc_vanilla_arm_template/_index.md:144]:
```suggestion
- Now that the first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using its public IP.
```

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_mi_arm_template/_index.md [hunk 2]
```diff
@@ -133,24 +135,26 @@ As mentioned, this deployment will leverage ARM templates. You will deploy a sin
 
     > **NOTE: The deployment time for this scenario can take ~15-20min**
 
-- Once Azure resources has been provisioned, you will be able to see it in Azure portal. At this point, the resource group should have **8 various Azure resources** deployed.
+- Once Azure resources has been provisioned, you will be able to see it in Azure portal. At this point, the resource group should have **8 various Azure resources** deployed (If you chose to deploy Azure Bastion, you will have **9 Azure resources**).
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_mi_arm_template/_index.md:138]:
```suggestion
- Once Azure resources have been provisioned, you will be able to see them in the Azure portal. At this point, the resource group should have **8 various Azure resources** deployed (If you chose to deploy Azure Bastion, you will have **9 Azure resources**).
```

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_mi_arm_template/_index.md [hunk 3]
```diff
@@ -133,24 +135,26 @@ As mentioned, this deployment will leverage ARM templates. You will deploy a sin
...
+- Once Azure resources have been provisioned, you will be able to see them in the Azure portal. At this point, the resource group should have **8 various Azure resources** deployed (If you chose to deploy Azure Bastion, you will have **9 Azure resources**).
 
     ![Screenshot showing ARM template deployment completed](./01.png)
 
     ![Screenshot showing the new Azure resource group with all resources](./02.png)
 
 ## Windows Login & Post Deployment
 
-- Now that first phase of the automation is completed, it is time to RDP to the client VM using it's public IP.
+- Now that first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using it's public IP.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_mi_arm_template/_index.md:146]:
```suggestion
- Now that the first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using its public IP.
```

File: docs/azure_arc_jumpstart/azure_arc_data/kubeadm/kubeadm_dc_vanilla_arm_template/_index.md [hunk 4]
```diff
@@ -148,24 +150,26 @@ As mentioned, this deployment will leverage ARM templates. You will deploy a sin
 
 ## Windows Login & Post Deployment
 
-Now that both the Ubuntu Kubernetes VM and the Windows Server client VM are created, it is time to login the Client VM.
-
-* Using it's public IP, RDP to the **Client VM**
+* Now that first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using it's public IP.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/kubeadm/kubeadm_dc_vanilla_arm_template/_index.md:153]:
```suggestion
* Now that the first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using its public IP.
```

File: docs/azure_arc_jumpstart/azure_arc_data/cluster_api/capi_azure_postgresql_hyperscale_arm_template/_index.md [hunk 5]
```diff
@@ -150,16 +152,18 @@ As mentioned, this deployment will leverage ARM templates. You will deploy a sin
 
 ## Windows Login & Post Deployment
 
-- Now that first phase of the automation is completed, it is time to RDP to the sidecar VM using it's public IP.
+- Now that first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using it's public IP.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/cluster_api/capi_azure_postgresql_hyperscale_arm_template/_index.md:155]:
```suggestion
- Now that the first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using its public IP.
```

File: docs/azure_arc_jumpstart/azure_arc_data/cluster_api/capi_azure_mssql_mi_arm_template/_index.md [hunk 6]
```diff
@@ -150,16 +152,18 @@ As mentioned, this deployment will leverage ARM templates. You will deploy a sin
 
 ## Windows Login & Post Deployment
 
-- Now that first phase of the automation is completed, it is time to RDP to the sidecar VM using it's public IP.
+- Now that first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using it's public IP.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/cluster_api/capi_azure_mssql_mi_arm_template/_index.md:155]:
```suggestion
- Now that the first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using its public IP.
```

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_postgresql_hyperscale_arm_template/_index.md [hunk 7]
```diff
@@ -133,24 +135,26 @@ As mentioned, this deployment will leverage ARM templates. You will deploy a sin
...
+- Once Azure resources has been provisioned, you will be able to see it in Azure portal. At this point, the resource group should have **8 various Azure resources** deployed (If you chose to deploy Azure Bastion, you will have **9 Azure resources**).
 
     ![Screenshot showing ARM template deployment completed](./01.png)
 
     ![Screenshot showing the new Azure resource group with all resources](./02.png)
 
 ## Windows Login & Post Deployment
 
-- Now that first phase of the automation is completed, it is time to RDP to the client VM using it's public IP.
+- Now that first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using it's public IP.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/aks/aks_postgresql_hyperscale_arm_template/_index.md:146]:
```suggestion
- Now that the first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using its public IP.
```

**Review by likamrat — CHANGES_REQUESTED** [review]
- Missing Bastion params the _azuredeploy.parameters.example.json_ of AKS, CAPI and MicroK8s

**Review by sebassem — COMMENTED** [review]
Fixed typos
Added parameters.example files for missing scenarios

**Review by likamrat — APPROVED** [review]
Awesome work!
LGTM

**Pull request merged**